arkane/encorr/isodesmic.py

```diff
@@ -76,35 +76,40 @@ def __init__(self, molecule, low_level_hf298, level_of_theory, high_level_hf298=
         if isinstance(molecule, Molecule):
             self.molecule = molecule
         else:
-            raise ValueError(f'ErrorCancelingSpecies molecule attribute must be an rmgpy Molecule object. Instead a '
-                             f'{type(molecule)} object was given')
+            raise ValueError(
+                f"ErrorCancelingSpecies molecule attribute must be an rmgpy Molecule object. Instead a " f"{type(molecule)} object was given"
```

Comment:
Are there any open pull requests (or other developers about to open them) touching arkane code?  A change to all the unit tests is disruptive enough - but there are many unrelated formatting changes to arkane in here too, that have nothing to do with unit tests.

Reply:
- #1899 is implementing new Arkane feature(s)
 - #1955 changes unit tests, but I saw your comment about that PR being confusing
 - #1967 is modifying tests and source
 - #2026 hits some arkane source
 - #2217 hits arkane source but seems abandoned